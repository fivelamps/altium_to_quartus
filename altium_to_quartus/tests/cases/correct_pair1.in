NODENAME DATA_P           $
    J2        72  U3       AF2
NODENAME DATA_N           $
    J2        70  U3       AE1