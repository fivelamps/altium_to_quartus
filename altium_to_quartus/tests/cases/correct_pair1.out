set_location_assignment PIN_AF2 -to "DATA"
set_location_assignment PIN_AE1 -to "DATA(n)"
